src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/IPluginContentsProcessor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.Toolkit.Plugins.Cli.Options;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Cli
+{
+    internal interface IPluginContentsProcessor
+    {
+        public ProcessedPluginContents Process(PackageGenCommonOptions options);
```

Comment:
if all options extend `PackageGenCommonOptions` would it make sense to have a base handler class instead of a new interface here?